api.js

```diff
@@ -1902,4 +1902,49 @@ API.prototype.ignoreApdex = function ignoreApdex() {
   transaction.ignoreApdex = true
 }
 
+/**
+ * Runs a function synchronously within a provided LLM custom attributes context and returns its return value.
+ *
+ * See documentation for newrelic.withLlmCustomAttributes for more information.
```

Comment:
Please add an `@see` attribute with an appropriate link.

Reply:
I maybe just remove this line for now.